components/tools/OmeroPy/src/omero/cli.py

```diff
@@ -1566,6 +1567,8 @@ def __call__(self, arg):
             needsForce = False
             for id in parts[1].split(","):
                 if "-" in id:
+                    warnings.warn("This flag is deprecated as of OMERO 5.3.2",
+                                  DeprecationWarning)
```

Comment:
What's deprecated, `--force` or  range specifier using `-`?

If it's `--force` then "deprecated" implies the new behaviour should work when actually the flag is still required.